week1/homework/src/server.js

```diff
@@ -10,6 +10,32 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    if (request.url === '/state') {
+      response.writeHead(200, { 'Content-Type': 'application/json' });
+      const main = { 'state': state };
```

Comment:
You don't need the variable, since you are only using it in `JSON.stringify()`, so you can do `JSON.stringify({'state':state})`